arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1936 @@
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/ExportDevicesRequest"
+          }
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobResponse"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/importDevices": {
+      "post": {
+        "tags": ["POST"],
+        "summary": "Import all the Devices in an IotHub",
+        "description": "Import all the Devices in an IotHub",
+        "operationId": "IotHubResource_ImportDevices",
+        "consumes": ["application/json", "text/json"],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "Name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "importDevicesRequest",
+          "in": "body",
+          "description": "The import devices request.",
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/ImportDevicesRequest"
+          }
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobResponse"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "Object": {
+      "type": "object",
+      "properties": {}
+    },
+    "SharedAccessSignatureAuthorizationRule": {
+      "description": "Contains properties that describe the keys to access the IotHub artifacts.",
+      "type": "object",
+      "properties": {
+        "keyName": {
+          "description": "Gets or sets the name of the key.",
+          "type": "string"
+        },
+        "primaryKey": {
+          "description": "Gets or sets the primary key.",
```

Comment:
You should just say `the ...` -- individual language generators will add the `Gets or sets` if it's appropriate (some languages don't support properties)